index.js

```diff
@@ -281,16 +313,26 @@ class LedgerBridgeKeyring extends EventEmitter {
   }
 
   _sendMessage (msg, cb) {
+    console.log("[LedgerBridgeKeyring][_sendMessage] message / callback is:", msg, cb)
     msg.target = 'LEDGER-IFRAME'
     this.iframe.contentWindow.postMessage(msg, '*')
     const eventListener = ({ origin, data }) => {
+      console.log("[LedgerBridgeKeyring][_sendMessage][Event received!]:", origin, data)
+
+      /*
       if (origin !== this._getOrigin()) {
+        console.error("[LedgerBridgeKeyring][_sendMessage][Event received!] bad origin:", origin, "/", this._getOrigin())
```

Comment:
Temporarily removing this check -- for some reason, the origin is being received as `metamask.github.io` despite me choosing `darkwing.github.io`